server/api/utils/clients/async_nuclio.py

```diff
@@ -18,35 +18,100 @@
 import aiohttp
 
 import mlrun.common.schemas
+import mlrun.errors
+import mlrun.runtimes.nuclio.api_gateway
 import mlrun.utils
 from mlrun.utils import logger
 
 NUCLIO_API_SESSIONS_ENDPOINT = "/api/sessions/"
-NUCLIO_API_GATEWAYS_ENDPOINT = "/api/api_gateways/"
+NUCLIO_API_GATEWAYS_ENDPOINT = "/api/api_gateways/{api_gateway}"
 API_GATEWAY_NAMESPACE_HEADER = "X-Nuclio-Api-Gateway-Namespace"
 NUCLIO_PROJECT_NAME_HEADER = "X-Nuclio-Project-Name"
+NUCLIO_PROJECT_NAME_LABEL = "nuclio.io/project-name"
+IGUAZIO_USERNAME_LABEL = "iguazio.com/username"
+MLRUN_CREATED_LABEL = "mlrun-created"
```

Comment:
These labels are not really nuclio-only, so I think the can be in a more common file (like `mlrun/common/constants.py`, or a new `mlrun/common/labels.py`.

Also, @theSaarco do we want to change `mlrun-created=True` to `created-by=mlrun`?

Reply:
@TomerShor moved it to `constants.py`. 

Do not resolve conversation until @theSaarco response.

Reply:
@rokatyy - The MLRun ticket [ML-4243](https://jira.iguazeng.com/browse/ML-4243) defines `mlrun/system-generated`, with potentially a `mlrun/system-generated-reason` (which I don't think is needed in this case).